alchemy-web/src/content/docs/providers/cloudflare/worker.mdx

```diff
@@ -818,6 +819,40 @@ only generate wrangler.json when you need to integrate with existing Wrangler-ba
 Alchemy's declarative approach eliminates the need for most wrangler.json configurations.
 :::
 
+### LogPush Configuration
```

Comment:
This should be higher up in the docs. Migration from wrangler to alchemy is lowest priority order I think. This belongs among the other features.

Reply:
🫡